ROOTFS/etc/waggle/sanity/warning/check_hanwha_camera.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+import sys
+import json
+import urllib.error, urllib.request
+import time
+
+def read_camera_manifest(path='/etc/waggle/node_manifest.json'):
+    with open(path, 'r') as file:
+        manifest = json.loads(file.read())
+        if 'cameras' not in manifest:
+            raise Exception(f'No cameras in the manifest: {path}')
+        else:
+            for camera in manifest['cameras']:
+                if camera['present'] == True:
+                    yield camera
+
+
+def http_request(url, user='waggle', pw='Waggle1@34', timeout=2):
```

Comment:
mmm... this is a public repo.  maybe we should pull the password from something more secure?  maybe /etc/waggle/config.ini? (maybe)

Reply:
The password used here is only for access to the camera and the account is restricted for admin access. I think it is fairly secure to be on public. @seanshahkarami suggested changing it to something random. I will probably update this file along with an update on re-provisioning of the cameras.

Reply:
@seanshahkarami The sanity check with the latest update returns "CAMERA_NOT_FOUND" error if any camera in the manifest not found. Also, "UNLISTED_CAMERA" error occurs when a camera responded to the checker is not in the manifest. Lastly, the json parse is covered with a try-catch. Thank you for the comments.